Packs/AsanaConnect/Integrations/AsanaConnect/AsanaConnect_description.md

```diff
@@ -0,0 +1,8 @@
+### Community Contributed Integration
+#### Integration Author: Cortex XSOAR
+Support and maintenance for this integration are provided by the author. Please use the following contact details:
+- **URL**: [https://www.paloaltonetworks.com/cortex](https://www.paloaltonetworks.com/cortex)
+***
+## Hello World
+- This section explains how to configure the instance of HelloWorld in Cortex XSOAR.
+- You can use the following API Key: `43ea9b2d-4998-43a6-ae91-aba62a26868c`
```

Comment:
This doesn't look right.
Please change the integration author and remove the hello world section.
You can read about description files here: https://xsoar.pan.dev/docs/documentation/integration-description